src/oracle/connection/stmt.rs

```diff
@@ -34,30 +33,8 @@ impl Statement {
             let place_holder = limit_clause.split_off(String::from("LIMIT ").len());
             mysql = mysql + &format!("OFFSET 0 ROWS FETCH NEXT {} ROWS ONLY", place_holder);
         }
-        let mut affected_table = "".to_string();
-        let mut is_returning = false;
-        if sql.starts_with("INSERT") || sql.starts_with("insert") {
-            if let Some(pos) = mysql.find("RETURNING") {
-                is_returning = true;
-//            mysql = mysql + &format!(" into ");
-                // determine the affected table, which is stupid, but works for `insert into #table ...` pretty well and is used as proof of concept
-                let mysqlcopy = mysql.clone();
-                let keywords : Vec<&str> = mysqlcopy.split(' ').collect();
-                affected_table = format!("{}", keywords[2]);
-
-                // we clone since we need the original statement
-                let mut _fields = mysql.split_off(pos + String::from("RETURNING").len());
-                // now that's just a shortcut to count the `,` which now come
-//            let single_fields : Vec<&str> = fields.split(',').collect();
-//            for i in 0..single_fields.len() {
-//                if i > 0 {
-//                    mysql = mysql + &format!(",");
-//                }
-//                mysql = mysql + &format!(":out{}", i);
-//            }
-                mysql = mysql + &format!(" rowidtochar(rowid) into :out1");
-            }
-        }
+//         let mut affected_table = "".to_string();
+        let  is_returning = (sql.starts_with("INSERT") || sql.starts_with("insert") ) && sql.contains("RETURNING");
```

Comment:
Please remark this, that it is error prone (we've seen it already crashing with migrations)